### PR TITLE
docs: Update install documentation for Ubuntu

### LINKED ISSUE #881: Update Ubuntu install doc for 17.10
Now that Docker have finally released the following:

- https://download.docker.com/linux/ubuntu/dists/artful/pool/stable/amd64/docker-ce_17.12.0~ce-0~ubuntu_amd64.deb

... we need to update and re-test the installation doc for Ubuntu 17.10:

- https://github.com/clearcontainers/runtime/blob/master/docs/ubuntu-installation-guide.md

### PR DESCRIPTION
**Pull request by GabyCT**
This will update the documentation for Ubuntu 17.10

Fixes #881

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by GabyCT** [comment]
@klynnrif could you please review it? thanks

File: docs/ubuntu-installation-guide.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Note:
 If you are installing on a system that already has Clear Containers 2.x
 installed, first read [the upgrading document](upgrading.md).
 
-Clear Containers **3.0** is available for Ubuntu\* **16.04** , **16.10** and **17.04**.
+Clear Containers **3.0** is available for Ubuntu\* **17.04** and **17.10**.
```
Comment (jodh-intel) [inline docs/ubuntu-installation-guide.md]: We need to retain 16.04 as that's an LTS release.

**Comment by GabyCT** [comment]
@jodh-intel ok :) changes applied

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
Hi @klynnrif - could you take a look at this (important :) one-liner?

File: docs/ubuntu-installation-guide.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Note:
 If you are installing on a system that already has Clear Containers 2.x
 installed, first read [the upgrading document](upgrading.md).
 
-Clear Containers **3.0** is available for Ubuntu\* **16.04** , **16.10** and **17.04**.
+Clear Containers **3.0** is available for Ubuntu\* **16.04** , **17.04** and **17.10**.
```
Comment (klynnrif) [inline docs/ubuntu-installation-guide.md]: Only suggest adding the final series comma :) for consistency per the style guide: Clear Containers **3.0** is available for Ubuntu\* **16.04** , **17.04**, and **17.10**.

**Comment by GabyCT** [comment]
@klynnrif changes applied thanks

**Comment by devimc** [comment]
lgtm

**Comment by devimc** [comment]
@GabyCT could you please also remove these two lines ?

```
$ sudo systemctl enable cc-proxy.socket
$ sudo systemctl start cc-proxy.socket
```

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by devimc** [comment]
@GabyCT thanks

**Pull request merged**